### ISSUE TITLE
Consider Linux builds for EL8/EL9

### ISSUE THREAD

**Issue by d1nuc0m** [VSCodium/vscodium#1951]
**Describe the bug**
Have you considered RPM builds for EL8/EL9? As of 2024/06/30, [RHEL7](https://endoflife.date/rhel) reached Extended Life Cycle support and [CentOS](https://endoflife.date/centos) 7 has gone EOL. 

**Please confirm that this problem is VSCodium-specific**
- [X] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

**Please confirm that the issue/resolution isn't already documented**
- [X] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/docs/index.md) and [the Troubleshooting page](https://github.com/VSCodium/vscodium/blob/master/docs/troubleshooting.md) and my issue is not mentioned there.

**To Reproduce**
N/A

**Expected behavior**
Have a build targeted to EL8/EL9

**Desktop (please complete the following information):**
 - OS: EL8/EL9
 - Architecture: independent
 - Version: independent
 - App Manager: YUM/DNF
 - Sandboxed: no

**Additional context**

- [VSCode RPMs](https://code.visualstudio.com/download) are for EL8
- EL7 builds work on EL8/EL9 (currently using it) but it'd be better to have a dedicated build

**Comment by sskamesh** [comment]
I wanted to create a separate issue for this, but it seems more relevant here. Adding to this, the 'el7' label in the file name (codium-1.91.1.24193-el7.x86_64.rpm) is misleading. When I try to install it on an EL7 system, it fails due to EL8's GLIBCXX 3.4x dependencies.

```
Loaded plugins: langpacks, versionlock
Examining codium-1.91.1.24193-el7.x86_64.rpm: codium-1.91.1.24193-el7.x86_64
Marking codium-1.91.1.24193-el7.x86_64.rpm to be installed
Resolving Dependencies
There are unfinished transactions remaining. You might consider running yum-complete-transaction, or "yum-complete-transaction --cleanup-only" and "yum history redo last", first to finish them. If those don't work you'll have to try removing/installing packages by hand (maybe package-cleanup can help).
--> Running transaction check
---> Package codium.x86_64 0:1.91.1.24193-el7 will be installed
--> Processing Dependency: libc.so.6(GLIBC_2.18)(64bit) for package: codium-1.91.1.24193-el7.x86_64
--> Processing Dependency: libstdc++.so.6(CXXABI_1.3.8)(64bit) for package: codium-1.91.1.24193-el7.x86_64
--> Processing Dependency: libstdc++.so.6(CXXABI_1.3.9)(64bit) for package: codium-1.91.1.24193-el7.x86_64
--> Processing Dependency: libstdc++.so.6(GLIBCXX_3.4.20)(64bit) for package: codium-1.91.1.24193-el7.x86_64
--> Processing Dependency: libstdc++.so.6(GLIBCXX_3.4.21)(64bit) for package: codium-1.91.1.24193-el7.x86_64
--> Processing Dependency: libstdc++.so.6(GLIBCXX_3.4.22)(64bit) for package: codium-1.91.1.24193-el7.x86_64
--> Finished Dependency Resolution
Error: Package: codium-1.91.1.24193-el7.x86_64 (/codium-1.91.1.24193-el7.x86_64)
           Requires: libstdc++.so.6(GLIBCXX_3.4.20)(64bit)
Error: Package: codium-1.91.1.24193-el7.x86_64 (/codium-1.91.1.24193-el7.x86_64)
           Requires: libstdc++.so.6(GLIBCXX_3.4.21)(64bit)
Error: Package: codium-1.91.1.24193-el7.x86_64 (/codium-1.91.1.24193-el7.x86_64)
           Requires: libc.so.6(GLIBC_2.18)(64bit)
Error: Package: codium-1.91.1.24193-el7.x86_64 (/codium-1.91.1.24193-el7.x86_64)
           Requires: libstdc++.so.6(CXXABI_1.3.8)(64bit)
Error: Package: codium-1.91.1.24193-el7.x86_64 (/codium-1.91.1.24193-el7.x86_64)
           Requires: libstdc++.so.6(CXXABI_1.3.9)(64bit)
Error: Package: codium-1.91.1.24193-el7.x86_64 (/codium-1.91.1.24193-el7.x86_64)
           Requires: libstdc++.so.6(GLIBCXX_3.4.22)(64bit)
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest

```

This is because Microsoft VS Code stopped supporting EL7 after version 1.85.2. Therefore, I suggest we could rename the package with 'el8' or, in my humble opinion, avoid the el(x) naming entirely.

// Kamesh.

**Comment by daiyam** [comment]
Yep, it isn't working on EL7 or Ubuntu18...

@sskamesh Do you need EL7 build?

**Comment by daiyam** [comment]
I've opened a discussion about EL7/Ubuntu18: https://github.com/VSCodium/vscodium/discussions/1969

**Comment by sskamesh** [comment]
Well, sure @daiyam =) I just thought 1.85.1 will be the end for EL7 hosts. Especially EL7 went EOL after June 2024. If we can get and keep the latest VSCodium for EL7/EL8/EL9, that'd be great.

**Comment by daiyam** [comment]
The latest version compatible with EL7 is `1.87.2.24072`. I've just tested it.
